### PR TITLE
Bravo improvements

### PR DESCRIPTION
**Pull request by YannRobert**
initially proposed in PR #122

I hope this fixes animation issues reported in #123

* improved animation timing (fade in + zoom in and out + sound now play much better)
* better performance and responsiveness
* the next round now only starts when the bravo animation and sound is fully finished
* the next round only start after a small delay (set to 5000 ms but you can fine tune it if this is not the appropriate value)
* timing can be tuned by just changing the private static constants at the beginning of the class. I tried to have meaningful variables and constants for readability

this PR depends on PR #125 (so it includes it)

### CODE REVIEW

**Comment by schwabdidier** [comment]
It is far better ! :) 

I was not clear last time certainly. I think it would be good if we see longer the animal image (in the context of where is the animal game) and during a shorten duration the bravo image.

Example : animal is found -> applauses start -> animal image goes big -> half a second or more then bravo image appears step by step -> end of animation -> new game

File: gazeplay/src/main/java/net/gazeplay/games/whereisit/WhereIsIt.java [hunk 1]
```diff
@@ -521,7 +521,7 @@ private void onCorrectCardSelected(WhereIsIt gameInstance) {
                 @Override
                 public void handle(ActionEvent actionEvent) {
 
-                    bravo.playWinTransition(scene, new EventHandler<ActionEvent>() {
+                    bravo.playWinTransition(scene, 200, new EventHandler<ActionEvent>() {
```
Comment (YannRobert) [inline gazeplay/src/main/java/net/gazeplay/games/whereisit/WhereIsIt.java]: here you can set a delay in order to display the winning card longer before the bravo animation starts

**Comment by YannRobert** [comment]
> I think it would be good if we see longer the animal image

I agree, but the applause sound duration is 6 seconds, so it's not easy to find the perfect timing.
Please see my comment https://github.com/schwabdidier/GazePlay/pull/126#pullrequestreview-85833015 for instruction on how to delay the bravo after the user found the winning card

> Example : animal is found -> applauses start -> animal image goes big -> half a second or more then bravo image appears step by step -> end of animation -> new game

This will need a change in the animation construction, because it is mixing a game animation (animal image goes big) and a generic animation (applauses start)
Also, this probably means the applauses sound will end before the graphical animation ends.

**Comment by schwabdidier** [comment]
I make a commit. I think I found a good trade-off between all parameters.

**Pull request merged**